ext/civicrm_admin_ui/managed/SavedSearch_Manage_Contribution_Pages.mgd.php

```diff
@@ -230,14 +230,15 @@
                 ],
                 [
                   'entity' => 'ContributionPage',
-                  'action' => 'delete',
+                  'action' => '',
                   'join' => '',
                   'target' => 'crm-popup',
                   'icon' => 'fa-trash',
                   'text' => E::ts('Delete'),
                   'style' => 'danger',
                   'path' => '',
                   'condition' => [],
+                  'task' => 'delete',
                 ],
               ],
               'type' => 'buttons',
```

Comment:
```suggestion
              'type' => 'menu',
```

since that seems to be what it doesn't like.